specifyweb/specify/api.py

```diff
@@ -252,6 +277,36 @@ def collection_dispatch(request, model) -> HttpResponse:
 
     return resp
 
+def collection_dispatch_bulk(request, model) -> HttpResponse:
+    """
+    Do the same as collection_dispatch, but for bulk POST operations.
+    Call this endpoint with a list of objects of the same type to create.
+    This reduces the amount of API calls needed to create multiple objects, like when creating multiple carry forwards.
```

Comment:
While not needed in this PR, maybe it would be worth extending this bulk operation endpoint to also include batch editing and deletion